doc/quickstart.md

```diff
@@ -0,0 +1,98 @@
+# A Quickstart Guide to Proper Benchmarking with BenchExec
+
+This guide provides a brief summary of instructions to set up reliable
+benchmark measurements using BenchExec and important points to consider. It is
+meant for users who either want to set up benchmarking from scratch or already
+have a simple setup using, e.g., `time`, `timeout`, `taskset`, `ulimit`, etc.
```

Comment:
I would prefer to rephrase this slightly. While I agree with the second half, I would say that users who do not have anything already should be encouraged to try out `benchexec` in many cases.

Something like the following maybe?

> It is meant for users who already have a basic setup using, e.g., `time`, `timeout`, `taskset`, `ulimit`, etc., or who want to try out BenchExec for small numbers of benchmarking runs. If you want to benchmark large number of runs or get the most out of what BenchExec provides as a benchmarking framework, consider using benchexec.

Reply:
I agree in spirit, but I would rather reformulate the subsequent text to indicate that there is more than `runexec`. So that the tutorial reads like "if you have something like this, use BenchExec. For a quick&dirty fix, simply replace these commands with `runexec`, for a proper solution, use `benchexec`"?